### PR TITLE
Fix kaniko-build-push tektonci job 🕹

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

There is no workspace in that Task…

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

/cc @afrittoli 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: tekton/ci/jobs.yaml [hunk 1]
```diff
@@ -139,21 +139,21 @@ spec:
     - /kaniko/executor
     - $(params.EXTRA_ARGS)
     - --dockerfile=$(params.DOCKERFILE)
-    - --context=$(workspaces.source.path)/$(params.CONTEXT)  # The user does not need to care the workspace and the source.
+    - --context=/workspace/source/$(params.CONTEXT)  # The user does not need to care the workspace and the source.
```
Comment (afrittoli) [inline tekton/ci/jobs.yaml:142]: this should be $(resources.inputs.source.path) instead, and same in the workingDir

Comment (vdemeester) [reply tekton/ci/jobs.yaml:142]: yes, and there is something else…

**Pull request merged**